src/watcher.js

```diff
@@ -255,14 +255,19 @@ class Watcher extends SafeEventEmitter {
             const currentChannel = twitch.getCurrentChannel();
             if (!currentChannel) return;
 
-            if (currentChannel.id === channel.id) return;
+            if (currentChannel.name === channel.name) return;
```

Comment:
The name is updated with the `load.chat` event. It's not always the case for the id.

Reply:
im not sure why this was changed or why it's needed